src/modules/LanItems.rb

```diff
@@ -485,6 +485,9 @@ def update_item_udev_rule!(based_on = :mac)
     # @param new_val     [string] value for new key
     # @return updated rule when replace_key is found, current rule otherwise
     def ReplaceItemUdev(replace_key, new_key, new_val)
+      Items()[@current]["udev"] = { "net" => [] } if !Items()[@current]["udev"]
```

Comment:
Other question is consider nil as a value to replace or not.. I mean, move that responsibility to an specific method.